minitest.el

```diff
@@ -274,6 +275,63 @@ Returns a (CMD . NAME) pair or nil."
   (and (buffer-file-name)
        (minitest-test-file-p (buffer-file-name))))
 
+;;;###autoload
+(defun minitest-toggle-test-and-target ()
+  "Switch to the test or the target file for the current buffer.
+If the current buffer is visiting a test file, switches to the
+target, otherwise the test."
+  (interactive)
+  (find-file (minitest--test-or-target)))
+
+(defun minitest--test-or-target ()
+  (if (minitest-buffer-is-test-p)
+      (minitest--target-file-for (buffer-file-name))
+    (minitest--test-file-for (buffer-file-name))))
+
+(defun minitest--test-file-for (a-file-name)
+  "Find test for the specified file."
+  (if (minitest-test-file-p a-file-name)
+      a-file-name
+    (let* ((replace-regex "^\\.\\./[^/]+/")
+           (test-directory (concat (minitest-project-root) "test"))
```

Comment:
what do you think about having a configurable test directory variable? it would be unconventional to have tests in a different directory, but i could see wanting to customize this with dir locals in an unconventional situation.

Reply:
that seems sensible. for what it's worth, `minitest--target-file-for` could maybe use a similar `defcustom` for these directories:

https://github.com/arthurnn/minitest-emacs/pull/59/files#diff-df48dc995d4017d19da30259503524fbd4b87931ab3f9b3fa05feba5a5189580R306

Reply:
just extracted these two variables. let me know if you think we should make any other changes!

Reply:
it looks great! i was coming up with some other unconventional cases but i don't want to block this for now.